src/RunState.h

```diff
@@ -24,12 +24,13 @@ extern void get_final_stats();
 extern void finish_run(int drain_events);
 extern void delete_run();	// Reclaim all memory, etc.
 extern void update_network_time(double new_network_time);
-extern void dispatch_packet(double t, const zeek::Packet* pkt,
-                            zeek::iosource::PktSrc* src_ps);
-extern void expire_timers(zeek::iosource::PktSrc* src_ps = nullptr);
+extern void dispatch_packet(zeek::Packet* pkt, zeek::iosource::PktSrc* pkt_src);
+extern void expire_timers();
 extern void zeek_terminate_loop(const char* reason);
 
-extern zeek::iosource::PktSrc* current_pktsrc;
+extern double check_pseudo_time(const Packet *pkt);
+
+extern zeek::iosource::PktSrc* current_pktsrc [[deprecated("Remove in v4.1. Use static_cast<zeek::iosource::PktSrc>(zeek::detail::iosource.)")]];
 extern zeek::iosource::IOSource* current_iosrc;
```

Comment:
How about adding a getter function for this one as well, and deprecated direct access to the global?

Reply:
I'm fine with that.